src/porepy/models/solution_strategy.py

```diff
@@ -247,6 +247,11 @@ def initial_condition(self) -> None:
         # Initialize time dependent ad arrays, including those for boundary values.
         self.update_time_dependent_ad_arrays()
 
+    @property
```

Comment:
This modification is consistent with `mypy` I believe with the code in general. Please check do you agree with this.

Reply:
From my understanding of properties, this is fine.